dali/pipeline/data/views.h

```diff
@@ -74,14 +75,21 @@ TensorShape<ndim> get_tensor_shape(const TensorList<Backend> &tl) {
   return convert_dim<ndim>(out);
 }
 
+
```

Comment:
This function does almost the same thing as the one with TensorVector, but it's written differently - I think these function should be as similar as they can be - and placed next to each other.

Reply:
done